src/coreComponents/fileIO/Outputs/VTKOutput.cpp

```diff
@@ -56,6 +56,11 @@ VTKOutput::VTKOutput( string const & name,
     setInputFlag( InputFlags::OPTIONAL ).
     setDescription( "Level detail plot. Only fields with lower of equal plot level will be output." );
 
+  registerWrapper( viewKeysStruct::numberOfTargetProcesses, &m_numberOfTargetProcesses ).
+    setApplyDefaultValue( MpiWrapper::commSize() ).
+    setInputFlag( InputFlags::OPTIONAL ).
+    setDescription( "Aggregate output data to be written." );
```

Comment:
I would expect something like:
```suggestion
    setDescription( "Number of output aggregate files to be written." );
```